### ISSUE TITLE
Unable to select neovim config with NVIM_APPNAME environment variable 

### ISSUE THREAD

**Issue by MattiaSilvestri** [kamiyaa/joshuto#540]
I use neovim as my default text editor, and I recently started to use the NVIM_APPNAME env variable to select between the neovim config I have in my config folder. I set an alias in my .zshrc file so that every time I type `vim` I get a specific config loaded up, like this: `alias vim="NVIM_APPNAME=custom_config nvim"`. However, I noticed that joshuto doesn't respect shell aliases or commands other than the binary of the app itself, making me unable to tell joshuto which config to use and forcing me to use the default one. Is there any way to solve this issue?

**Comment by kamiyaa** [comment]
Yeah, `alias` is a shell concept so joshuto won't really know about it without reading `.bashrc`, `.zshrc` etc.

One way to fix this is to add a shell script to your path.
ie.

vim
```
#/bin/bash

NVIM_APPNAME=custom_config nvim
```

and then place it inside one of your path directories.
Then joshuto will be able to see it
